pkg/apis/storage/util/helpers.go

```diff
@@ -26,6 +26,9 @@ const IsDefaultStorageClassAnnotation = "storageclass.kubernetes.io/is-default-c
 // TODO: remove Beta when no longer used
 const BetaIsDefaultStorageClassAnnotation = "storageclass.beta.kubernetes.io/is-default-class"
 
+// AlphaIsDefaultVolumeAttributesClassAnnotation is the alpha version of IsDefaultVolumeAttributesClassAnnotation.
+const AlphaIsDefaultVolumeAttributesClassAnnotation = "volumeattributesclass.alpha.kubernetes.io/is-default-class"
```

Comment:
`"volumeattributesclass.alpha.kubernetes.io/is-default-class"` or `"volumeattributesclass.kubernetes.io/is-default-class"` ?

@sunnylovestiramisu

Reply:
I am trying to find examples but I cannot find any, @jsafrane did you use alpha for storage class?